msteams-platform/concepts/device-capabilities/location-capability.md

```diff
@@ -10,54 +10,58 @@ ms.author: surbhigupta
 
-> [!NOTE]
-> Currently, Microsoft Teams support for location capabilities is available for mobile clients only.
+* Update the app manifest file and call the APIs.
+* Have working knowledge of [code snippets](#code-snippets) for calling the location APIs.
+* Handle errors in your Teams app with the help of [API response errors](#error-handling).
 
 ## Update manifest
 
-Update your Teams app [manifest.json](../../resources/schema/manifest-schema.md#devicepermissions) file by adding the `devicePermissions` property and specifying `geolocation`. It allows your app to ask for requisite permissions from users before they start using the location capabilities. The update for app manifest is as follows:
+Update your Teams app [manifest.json](../../resources/schema/manifest-schema.md#devicepermissions) file by adding the `devicePermissions` property and specifying `geolocation`. It allows your app to ask for requisite permissions from users before they start using the location  capabilities
+. The update for app manifest is as follows:
 
 ``` json
 "devicePermissions": [
     "geolocation",
 ],
 ```
 
-> [!NOTE]
-> * The **Request Permissions** prompt is automatically displayed when a relevant Teams API is initiated. For more information, see [request device permissions](native-device-permissions.md).
-> * Device permissions are different in the browser. For more information, see [browser device permissions](browser-device-permissions.md).
+> **Note** </br>
+• The request permissions prompt is automatically displayed when a relevant Teams API is initiated. For more information, see [request device permissions](native-device-permissions.md).</br>
+• The device permissions are different in the browser. For more information, see [browser device permissions](browser-device-permissions.md).
+
 
 ## Location APIs
 
 You must use the following set of APIs to enable your device's location capabilities:
 
-| API      | Description   |
-| --- | --- |
-|[getLocation](/javascript/api/@microsoft/teams-js/microsoftteams.location?view=msteams-client-js-latest#getLocation_LocationProps___error__SdkError__location__Location_____void_&preserve-view=true) | Gives user’s current device location or opens native location picker and returns the location chosen by the user. |
-|[showLocation](/javascript/api/@microsoft/teams-js/microsoftteams.location?view=msteams-client-js-latest#showLocation_Location___error__SdkError__status__boolean_____void_&preserve-view=true) | Shows location on map. |
+| API      | Description |Input configuration `allowChooseLocation` |Input configuration `showMap` |
+| --- | --- |--- |--- |
+|`getLocation`| Gives user’s current device location or opens native location picker and returns the location chosen by the user. |True: Users can choose any location of their choice.</br> False: users cannot change their current location |False: The current location is fetched without displaying the map. | 
```

Comment:
Provides user’s current device location or opens native location picker and returns the location chosen by the user.